.gitignore

```diff
@@ -32,4 +32,5 @@ _build
 _work
 
 *.tgz
-charts/operator/crds
\ No newline at end of file
+charts/operator/crds
+charts/operator/deviceplugin_v1_sgxdeviceplugin.yaml
```

Comment:
how do I modify these parameters using helm?

Reply:
each entry is basically a config entry so all of them should be configurable. I also had a chat with @poussa  and he proposed to make these "included" based on a boolean, e.g. `{{- if .Values.devices.SGX | empty | not }}`...

Reply:
I've pushed a fix to this, to test installation:

`helm install charts/operator  --debug  --dry-run  --generate-name --set sgx.enabled=true`

Reply:
looks very good to me! one question: can this `<device>.enabled=true` be easily taken to the operators' parameters like this too: https://github.com/intel/intel-device-plugins-for-kubernetes/blob/23eacb3188df5d753792433b73c646611f272d32/deployments/operator/device/sgx/sgx.yaml#L13

Reply:
@rmstoi this chart does not deploy anything:

```
$ kubectl get pods -n default
No resources found in default namespace.
$ helm status operator-1651210270
NAME: operator-1651210270
LAST DEPLOYED: Fri Apr 29 08:31:10 2022
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing intel-device-plugins-operator.
```

Reply:
> looks very good to me! one question: can this `<device>.enabled=true` be easily taken to the operators' parameters like this too:
> 
> https://github.com/intel/intel-device-plugins-for-kubernetes/blob/23eacb3188df5d753792433b73c646611f272d32/deployments/operator/device/sgx/sgx.yaml#L13

I don't see why not

Reply:
> @rmstoi this chart does not deploy anything:

I pushed a fix to this, with a new templates/operator.yaml originally created by kustomize